File: practice questions/Arrays/IsSorted.java
class IsSorted{
    public boolean isSorted(int[] arr)
    {
        
        for(int i=1;i<arr.length;i++){
            if(arr[i-1]>arr[i]){
                return false;
            }
        }
        return true;
    }
    public static void main(String[] args) {
        IsSorted is= new IsSorted();
        int[] arr = {2,3,4,5,6,7};
        boolean res=is.isSorted(arr);
        System.out.println(res);
    }
}

File: practice questions/Arrays/issorted.py
def isSorted(arr : list) -> bool:
    for i in range(1,len(arr)):
        if arr[i-1]>arr[i]:
            return False
    return True

if __name__=="__main__":
    arr=[1,2,3,4,5,6,9,7]
    print(isSorted(arr))
    
